bundles/org.openhab.binding.vitotronic/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -528,6 +529,13 @@
 		<state pattern="%d" readOnly="false" max="4" min="0" step="1"/>
 	</channel-type>
 
+	<channel-type id="currentmode">
+		<item-type>Number</item-type>
+		<label>Currentmode</label>
+		<description>Currentmode</description>
```

Comment:
Description to be removed if it adds no value to the label.
At least, use correct English, that is current mode (2 words).